frontend/e2e/exercises-page.ts

```diff
@@ -52,4 +52,20 @@ export class ExercisesPage {
     async checkTagIsVisible(tag: string): Promise<void> {
         await expect(this.page.locator(`app-exercises-listing mat-chip-list mat-chip:has-text("${tag}")`).first()).toBeVisible();
     }
+
+    async checkNextPageButtonIsUsable(): Promise<void> {
+        await expect(this.page.locator('.mat-paginator-range-actions button.mat-paginator-navigation-next').first()).not.toBeDisabled;
```

Comment:
**question:** do these actually work correctly since there's no parenthesis in the end?